src/vis/vis.js

```diff
@@ -315,8 +315,21 @@ var VisModel = Backbone.Model.extend({
   },
 
   reload: function (options) {
-    options = _.extend({includeFilters: true}, options);
-    options = _.pick(options, 'sourceId', 'forceFetch', 'success', 'error', 'includeFilters');
+    options = options || {};
+    var successCallback = options.success;
+    var errorCallback = options.error;
+
+    options = _.extend({
+      includeFilters: true,
+      success: function () {
+        this.trigger('reloaded');
+        successCallback && successCallback();
+      }.bind(this),
+      error: function () {
```

Comment:
We're not currently receiving any error here, so there's no error to "return"...